venv/ParsingClass.py

```diff
@@ -0,0 +1,242 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+#Parsing Class
+#Class returrning parseable file after storing in datafrane datastrucuture
+class Parsing():
+
+
+    def __init__(self,year):
+
+        self.year = year
+
+
+    def perform_parsing(self):
+
+        if self.year == "2004":
+             self.result = print("I read year 2004")
+             v41 = pd.read_csv("weatherfiles/Murree_weather_2004_Aug.txt")
+             v42 = pd.read_csv("weatherfiles/Murree_weather_2004_Dec.txt")
+             v43 = pd.read_csv("weatherfiles/Murree_weather_2004_Jul.txt")
+             v44 = pd.read_csv("weatherfiles/Murree_weather_2004_Jun.txt")
```

Comment:
Why don't you let the program do this work? Why write read line for each and every file your self?

Construct the file path based on user input. Store list of months in a separate variable. Concatenate month and year using using f-strings.